### PR TITLE
Install 'iputils' for CC test case `ebtables_NETFILTER_PKT FAIL`

### PR DESCRIPTION
**Pull request by Amrysliu**
Please see the individual commit message
Related: https://progress.opensuse.org/issues/98856, https://progress.opensuse.org/issues/101644, https://progress.opensuse.org/issues/101920
Verify run: https://openqa.suse.de/tests/7604230# (s390)
                 https://openqa.suse.de/tests/7604488# (aarch64)
                 https://openqa.suse.de/tests/7604235# (64bit)

### CODE REVIEW

**Comment by lilyeyes** [comment]
Could you please add more information to poo or code or PR why the fix can work?

**Comment by lilyeyes** [comment]
Any explanation to the failure on https://openqa.suse.de/tests/7594890#step/ip_eb_tables/34?

File: tests/security/cc/ip_eb_tables.pm [hunk 1]
```diff
@@ -24,6 +24,9 @@ sub run {
 
     select_console 'root-console';
 
+    # install command ping
```
Comment (lilyeyes) [inline tests/security/cc/ip_eb_tables.pm:27]: Coding style consistence: s/install/Install/ :)

File: tests/security/cc/ip_eb_tables.pm [hunk 2]
```diff
@@ -24,6 +24,9 @@ sub run {
...
+    zypper_call('in iputils');
+
     # Configure bridge for ip_eb_tables workaround
```
Comment (rfan1) [inline tests/security/cc/ip_eb_tables.pm:30]: Xiaojing,
Bridge setup is already there for x86-64 and aarch64, can you please enhance this script as well?
Any, you can file a new PR for it.

File: tests/security/cc/ip_eb_tables.pm [hunk 3]
```diff
@@ -24,6 +24,9 @@ sub run {
 
     select_console 'root-console';
 
+    # install command ping
+    zypper_call('in iputils');
```
Comment (rfan1) [inline tests/security/cc/ip_eb_tables.pm:28]: I'd suggest, you add this into cc_setup test module. iputils should be used by audit-remote test as well. I would consider it a "basic" cmd :)

**Comment by lilyeyes** [comment]
why crypto still failed on "make -m64"?

**Comment by jouyingbin** [comment]
There is a typo is your 2nd commit. 

s/aviod/avoid

**Comment by Amrysliu** [comment]
> Could you please add more information to poo or code or PR why the fix can work? E.g., fail_safe is failed on execution time_out another one is lack of pkgs E.g., past the important error mesg to poo.

Added more details in the commit message.

**Comment by Amrysliu** [comment]
> why crypto still failed on "make -m64"?

I didn't rebase my branch, so it didn't include this fix. Have done it.

**Comment by Amrysliu** [comment]
> There is a typo in your 2nd commit. ( Increase timeout when running fail_safe)
> 
> **s/aviod/avoid**

fixed, Thanks a lot.

File: tests/security/cc/ip_eb_tables.pm (hunk 3 above)
Comment (Amrysliu) [reply tests/security/cc/ip_eb_tables.pm]: Done.

File: tests/security/cc/ip_eb_tables.pm (hunk 2 above)
Comment (Amrysliu) [reply tests/security/cc/ip_eb_tables.pm]: Did in another commit

**Comment by Amrysliu** [comment]
> Any explanation to the failure on https://openqa.suse.de/tests/7594890#step/ip_eb_tables/34?

Here is a bug for it: https://bugzilla.suse.com/show_bug.cgi?id=1192324

**Review by lilyeyes — APPROVED** [review]
LGTM.

File: tests/security/cc/ip_eb_tables.pm [hunk 4]
```diff
@@ -24,27 +25,33 @@ sub run {
 
     select_console 'root-console';
 
-    # Configure bridge for ip_eb_tables workaround
-    assert_script_run("cat > $f_ifcfg_br0 <<'END'\n$br0_config\nEND\n( exit \$?)");
+    # In system role is CC, the bridge is set up automatically in x86 and arm,
```
Comment (rfan1) [inline tests/security/cc/ip_eb_tables.pm:28]: Change to:
audit ebtables tests needs bridge network, In cc system role....

File: tests/security/cc/ip_eb_tables.pm [hunk 5]
```diff
@@ -24,27 +25,33 @@ sub run {
...
+    # so we only need to configure the bridge network for s390
```
Comment (jouyingbin) [inline tests/security/cc/ip_eb_tables.pm:29]: it should be **s390x** here

Comment (jouyingbin) [reply tests/security/cc/ip_eb_tables.pm:29]: In commit message should also be changed to **s390x** too.

File: tests/security/cc/ip_eb_tables.pm (hunk 4 above)
Comment (Amrysliu) [reply tests/security/cc/ip_eb_tables.pm]: Done

File: tests/security/cc/ip_eb_tables.pm (hunk 5 above)
Comment (Amrysliu) [reply tests/security/cc/ip_eb_tables.pm]: fixed

**Review by rfan1 — APPROVED** [review]
LGTM

**Review by jouyingbin — APPROVED** [review]
LGTM, thanks for the change.

**Pull request merged**